### ISSUE TITLE
Feature request: More "Forced Yes" scenarios

### ISSUE THREAD

**Issue by Lynvae** [daemitus/YesAlready#78]
_Forced Yes_ should work on materia extraction through the Materia Creation window. Allow the _Forced Yes_ setting to also default to Confirm on confirm/cancel windows, or add that as an optional setting

**Comment by Lynvae** [comment]
Correct! I don't have any spiritbonded gear so I can't get the exact text right now, but including the dialogue box that pops up in the "forced yes" setting would be wonderful

**Comment by guppyfaced** [comment]
Please see the template at the wiki page, which I think already has that action configured. Disable the actions you don't want.

https://github.com/daemitus/YesAlready/wiki/Commonly-used-%22Yes-Already%22-configuration

**Comment by Lynvae** [comment]
What I'm looking for is the inclusion of the dialogue in the "Forced Yes" setting, when I hold the hotkey assigned to that. I don't want it to always select yes by default

**Comment by Lynvae** [comment]
![Spiritbond](https://user-images.githubusercontent.com/13019124/187031448-86ad287e-f497-4fb8-81e3-60a9cda0c364.png)
Got some gear to 100% spiritbond so I could get this screenshot. If this dialogue could be included in the forced yes hotkey

**Comment by guppyfaced** [comment]
There's a setting for that in Bothers, under Materia Retrieve Dialog

**Comment by Lynvae** [comment]
I know there is, but I'd like for it to be included in the forced yes hotkey. If that's an unreasonable request I'll manage with toggling the permanent setting, but having the dialogue skipped by holding my hotkey would be preferable

**Comment by guppyfaced** [comment]
I think that setting only affects materia extraction. Why would you want to toggle that on/off?

**Comment by guppyfaced** [comment]
Just out of curiousity
